ui/ui.js

```diff
@@ -252,6 +252,7 @@ shaka.ui.Overlay = class {
       fullScreenElement: this.videoContainer_,
       preferDocumentPictureInPicture: true,
       showAudioChannelCountVariants: true,
+      showAudioCodecs: true,
```

Comment:
@joeyparrish Do we want this enabled by default? I think that the default value should be false.

Reply:
OK, I disabled it by default. I've done it mostly to simplify manual testing of this change.